samples/zephyr/sysbuild/hello_world/Kconfig.sysbuild

```
@@ -0,0 +1,7 @@
+# Copyright (c) 2025 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+source "${ZEPHYR_BASE}/share/sysbuild/Kconfig"
+
+config REMOTE_BOARD
+	string "The board used for remote target"
```

Why is `REMOTE_BOARD` having a prompt ?

This should default to a sane value based on the SoC.
Like this:
```suggestion
config REMOTE_BOARD
	string
	default "$(BOARD)/nrf54lv10dk/nrf54lv10a/cpuflpr" if SOC_NRF54LV10A__ENGA_CPUAPP
```
or similar.
```

---

Then this is a topic for different, new PR, regarding what should be nrf official way to enable 2 cores with sysbuild.
I do not want to mix changing this with simple adding new platform to existing testing.